modules/cnf-topology-aware-lifecycle-manager-troubleshooting.adoc

```diff
@@ -469,3 +469,31 @@ $ oc describe pod -n openshift-talo-pre-cache <pod name>
 ----
 $ oc describe job -n openshift-talo-pre-cache pre-cache
 ----
+
+[discrete]
+[id="talo-troubleshooting-pre-placement-tolerations_{context}"]
+=== Matching policies and `ManagedCluster` CRs before the managed cluster is available
+
+Issue:: You want {rh-rhacm} to match policies and managed clusters before the managed clusters become available.
++
+Resolution:: To ensure that {cgu-operator} correctly applies the {rh-rhacm} policies specified in the `spec.managedPolicies` field of the `ClusterGroupUpgrade` (CGU) CR, {cgu-operator} needs to match these policies to the managed cluster before the managed cluster is available. 
+The {rh-rhacm} `PolicyGenerator` uses the generated `Placement` CR to do this automatically. 
+By default, this `Placement` CR includes the necessary tolerations. 
+This ensures proper {cgu-operator} behavior. 
++
+The expected `spec.tolerations` settings in the `Placement` CR are:
++
+[source,terminal]
+----
+  tolerations:
+    - key: cluster.open-cluster-management.io/unavailable
+     operator: Exists
+    - key: cluster.open-cluster-management.io/unreachable
+     operator: Exists
+----
++
+If you use a custom `Placement` CR instead of the one generated by the {rh-rhacm} `PolicyGenerator`, you should include these tolerations in that `Placement` CR.
```

Comment:
```suggestion
If you use a custom `Placement` CR instead of the one generated by the {rh-rhacm} `PolicyGenerator`, include these tolerations in that `Placement` CR.
```
ISG Word usage guidance about "should": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#should